dotCMS/src/main/java/com/dotmarketing/quartz/job/DistributionJournalCleanup.java

```diff
@@ -1,47 +1,30 @@
 package com.dotmarketing.quartz.job;
 
+import com.dotcms.business.WrapInTransaction;
 import java.util.Date;
 
 import org.quartz.Job;
 import org.quartz.JobExecutionContext;
 import org.quartz.JobExecutionException;
 
 import com.dotmarketing.business.APILocator;
-import com.dotmarketing.db.DbConnectionFactory;
-import com.dotmarketing.db.HibernateUtil;
-import com.dotmarketing.exception.DotHibernateException;
 import com.dotmarketing.util.Logger;
 
 public class DistributionJournalCleanup implements Runnable, Job {
 
     public DistributionJournalCleanup() {
     }
 
+	@WrapInTransaction
     public void run() 
     {
     	try 
     	{
-    		HibernateUtil.startTransaction();
     		APILocator.getDistributedJournalAPI().processJournalEntries();
-    		HibernateUtil.closeAndCommitTransaction();
     	}
     	catch (Exception e) 
```

Comment:
I cannot remove the try/catch because the method processJournalEntries throws DotDataException... and I cannot change the run method signature.
So i will keep the try-catch but will re-throw as RuntimeException instead.